parallel-consumer-core/src/main/java/io/confluent/csid/utils/StringUtils.java

```diff
@@ -9,7 +9,7 @@
 public class StringUtils {
 
     public static String msg(String s, Object... args) {
-        return MessageFormatter.basicArrayFormat(s, args);
+        return MessageFormatter.arrayFormat(s, args).getMessage();
```

Comment:
@astubbs : Was it a conscious choice to use `basicArrayFormat`? That method expects that a throwable is _not_ the last element in the array. If so, `arrayFormat(s, args, null).getMessage()` is the equivalent in 1.7.30 as far as I can see (same as `basicArrayFormat` does internally)

```
    /**
     * Assumes that argArray only contains arguments with no throwable as last element.
     */
    final public static String basicArrayFormat(final String messagePattern, final Object[] argArray) {
    	FormattingTuple ft = arrayFormat(messagePattern, argArray, null);
    	return ft.getMessage();
    }
```

Reply:
cool, yeah let's get rid of the pre-release dependencies.